lib/perl/Genome/Sys/Lock/NessyBackend.pm

```diff
@@ -178,6 +181,32 @@ sub min_timeout {
     return 5;
 }
 
+sub _translate_resource {
+    my ($self, $resource) = @_;
+
+    my @subpath_parts;
+    my $resource_is_subpath = 1;
+
+    my @base_parts = File::Spec->splitdir($ENV{GENOME_LOCK_DIR});
+    my @resource_parts = File::Spec->splitdir($resource);
+    my $zip_iter = List::MoreUtils::each_array(@base_parts, @resource_parts);
```

Comment:
Yeah, I only did it because it matches the example in the `List::MoreUtils` docs..